### PR TITLE
Adds atomized key support for `decode/2` and `decode!/2`

### LINKED ISSUE #2: Add option to parse to atomized keys when decoding a yaml

### PR DESCRIPTION
**Pull request by the-pat**
Adds a keyword list to `decode/2` and `decode!/2` for the user to optionally pass `[atoms: true]` to convert the map keys into atoms (rather than strings).

Thanks for creating this project, @GPrimola! Happy Hacktober! 👻🎃 

Closes #2

### CODE REVIEW

**Comment by GPrimola** [comment]
Hey @the-pat!

What a beautiful code! Thank you very much for your contribution! ❤️

File: lib/yamel.ex [hunk 1]
```diff
@@ -6,13 +6,25 @@ defmodule Yamel do
   @type t :: map() | list()
   @type yaml :: binary()
 
-  @spec decode!(yaml()) :: Yamel.t()
-  defdelegate decode!(yaml_string), to: YamlElixir, as: :read_from_string!
+  @spec decode!(yaml(), atoms: boolean()) :: Yamel.t()
+  def decode!(yaml_string, options \\ []) do
+    atoms = Keyword.get(options, :atoms)
```
Comment (GPrimola) [inline lib/yamel.ex]: @the-pat I'll just ask you to use`keys: :atoms` as this options. The reason for that is so the interface of `Yamel` is as close as possible to [`Jason`](https://github.com/michalmuskala/jason).

**Review by GPrimola — CHANGES_REQUESTED** [review]

File: lib/yamel.ex (hunk 1 above)
Comment (the-pat) [reply lib/yamel.ex]: No problem! I'll hop on that.

**Comment by GPrimola** [comment]
Fantastic @the-pat !! Thank you very much!! 🎉

**Review by GPrimola — APPROVED** [review]

**Pull request merged**